blocks/library/latest-comments/index.js

```diff
@@ -0,0 +1,209 @@
+
+				this.latestCommentsRequest
+					.then( latestComments => this.setState( { latestComments } ) );
+
+				setAttributes( { commentsToShow: commentsToShowNext } );
+			}
+		}
+
+		toggleDisplayAvatar() {
+			const { displayAvatar } = this.props.attributes;
+			const { setAttributes } = this.props;
+
+			setAttributes( { displayAvatar: ! displayAvatar } );
+		}
+
+		toggleDisplayExcerpt() {
+			const { displayExcerpt } = this.props.attributes;
+			const { setAttributes } = this.props;
+
+			setAttributes( { displayExcerpt: ! displayExcerpt } );
+		}
+
+		toggleDisplayTimestamp() {
+			const { displayTimestamp } = this.props.attributes;
+			const { setAttributes } = this.props;
+
+			setAttributes( { displayTimestamp: ! displayTimestamp } );
+		}
+
+		changeCommentsToShow( commentsToShow ) {
+			const { setAttributes } = this.props;
+
+			setAttributes( { commentsToShow: parseInt( commentsToShow, 10 ) || 0 } );
+		}
+
+		render() {
+			const { latestComments } = this.state;
+			const { setAttributes } = this.props;
+
+			if ( ! latestComments.length ) {
+				return (
+					<Placeholder
+						icon="admin-post"
+						label={ __( 'Latest Comments' ) }
+					>
+						<Spinner />
+					</Placeholder>
+				);
+			}
+
+			const { focus } = this.props;
+			const { align, displayAvatar, displayTimestamp, displayExcerpt } = this.props.attributes;
+
+			return [
+				focus && (
+					<BlockControls key="controls">
+						<BlockAlignmentToolbar
+							value={ align }
+							onChange={ ( nextAlign ) => {
+								setAttributes( { align: nextAlign } );
+							} }
+							controls={ [ 'left', 'center', 'right', 'wide', 'full' ] }
+						/>
+					</BlockControls>
+				),
+				focus && (
+					<InspectorControls key="inspector">
+						<BlockDescription>
+							<p>{ __( 'Shows a list of your site\'s most recent comments.' ) }</p>
+						</BlockDescription>
+						<h3>{ __( 'Latest Comments Settings' ) }</h3>
+
+						<ToggleControl
+							label={ __( 'Display avatar' ) }
+							checked={ displayAvatar }
+							onChange={ this.toggleDisplayAvatar }
+						/>
+
+						<ToggleControl
+							label={ __( 'Display timestamp' ) }
+							checked={ displayTimestamp }
+							onChange={ this.toggleDisplayTimestamp }
+						/>
+
+						<ToggleControl
+							label={ __( 'Display excerpt' ) }
+							checked={ this.props.attributes.displayExcerpt }
+							onChange={ this.toggleDisplayExcerpt }
+						/>
+
+						<TextControl
+							label={ __( 'Number of comments to show' ) }
+							type="number"
+							min={ MIN_COMMENTS }
+							max={ MAX_COMMENTS }
+							value={ this.props.attributes.commentsToShow }
+							onChange={ ( value ) => this.changeCommentsToShow( value ) }
+						/>
+					</InspectorControls>
+				),
+				<ul className={ this.props.className } key="latest-comments">
+					{ latestComments.map( ( comment, i ) =>
+						<li key={ i }>
+							{ displayAvatar && comment.author_avatar_urls[ 96 ] &&
```

Comment:
No, it's not safe because `rest_avatar_sizes` can be filtered to cause different sizes to be returned in the response. So yeah, using the largest size I think is the right move.